src/main/java/uk/gov/pay/adminusers/model/PatchRequest.java

```diff
@@ -26,6 +26,10 @@ public static PatchRequest from(JsonNode node) {
         return new PatchRequest(node.get("op").asText(), node.get("path").asText(), node.get("value").asText());
     }
 
+    public String getOp() {
+        return op;
+    }
+    
```

Comment:
what breakage does adding this getter method fix?

Reply:
It stops PMD warning about an unused private field.